core/string/translation.cpp

```diff
@@ -923,6 +933,9 @@ void Translation::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("_set_messages"), &Translation::_set_messages);
 	ClassDB::bind_method(D_METHOD("_get_messages"), &Translation::_get_messages);
 
+	GDVIRTUAL_BIND(_get_plural_message, "src_text", "plural_text", "n", "context");
+	GDVIRTUAL_BIND(_get_message, "src_text", "context");
```

Comment:
Should match the non virtual arguments so that documentation is easier to parse for both methods.
```suggestion
	GDVIRTUAL_BIND(_get_plural_message, "src_message", "src_plural_message", "n", "context");
	GDVIRTUAL_BIND(_get_message, "src_message", "context");
```

Reply:
Fixed.